specification/2.0/appendix/performance-metrics/README.md

```diff
@@ -0,0 +1,77 @@
+# Runtime Performance Metrics 
+
+The goal of these metrics is to asses the runtime performance needs of glTF assets.  
+By runtime we mean the cpu/gpu, bandwidth and memory resources needed to render the model in realtime.  
+These performance metrics are divided in two major categories - `complexity` and `memory`
+
+## Performance Metric Categories
+
+### Complexity  
+
+The `complexity`category provides metrics to calculate cpu/gpu and memory bandwidth target requirements - larger numbers will mean a more powerful target device is needed.  
+These metrics do not provide a mapping to target device performance, this mapping must be done by users of the metrics.  
+
+### Memory  
+
+The `memory` category provides metrics to calculate the runtime memory (size) needs of the model.  
+Using these metrics an estimate of target memory can be calculated - however keep in mind that the exact memory requirements will vary with viewer implementations.  
+
+
+
+|                 |             |          |                                                                | 
+|-----------------|-------------|----------|----------------------------------------------------------------|
+| Name            | Category    | Property |Description                                                     |
+|[vertexcount]    | Complexity  |Scene     |Total number of vertices used by a model in a scene             |  
+|[nodecount]      | Complexity  |Scene     |Max nodecount in scene (add upp all nodes in a scene)           |  
+|[primitivecount] | Complexity  |Scene     |Total number of referenced primitives (per scene).  This figure is the un-batched number of primitives, engines may optimize if primitives and meshes share textures. |  
+|[textures]       | Complexity  |Scene     |Flags specifying presence of materials and it's texture usage, this is the aggregated max usage. BASECOLOR, METALLICROUGHNESS, NORMAL, OCCLUSION, EMISSIVE, (SPECULARGLOSS) |  
+|[attributes]     | Memory      | Asset    | The total memory footprint, in bytes, for the models attribute buffer storage |  
+|[texturesizes]  | Memory      | Asset    | The sizes of defined textures in the Asset. |  
+
```

Comment:
Either the table or the JSON description below should indicate the *type* of the properties. E.g. `integer` or `number`.

Reply:
Added